src/sentry/conf/server.py

```diff
@@ -666,8 +680,8 @@ def create_partitioned_queues(name):
             'level': 'INFO',
         },
         'django.request': {
-            'level': 'ERROR',
-            'handlers': ['console'],
+            'level': 'WARNING',
```

Comment:
Won't this mean `django.request` will now log at `WARNING` to `console` too?

I'm not sure how noisy that is or if anyone would care... just making sure we're OK with it.

Reply:
Yeah, would be pretty simple to create a `console.error` log handler that filters back down to only `ERROR` and above if it's a problem. TBH I'm not really sure if console messages get collected anywhere else so not sure what the constraints are on how noisy it can be.